src/Tribe/Admin/Help_Page.php

```diff
@@ -66,6 +66,32 @@ public function is_current_page() {
 		return in_array( $current_screen->id, $help_pages );
 	}
 
+	/**
+	 * Checks if the current page is the TEC Help one
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_tec_events_help_page() {
+		global $current_screen;
+
+		return 'tribe_events_page_tec-events-help' === $current_screen->id;
+	}
+
+	/**
+	 * Checks if the current page is the Tickets Help one
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_tec_tickets_help_page() {
+		global $current_screen;
+
+		return 'tickets_page_tec-tickets-help' === $current_screen->id;
```

Comment:
Lets with adding conditionals to prevent weird warnings if this is executed in the wrong spot.

When checking `global $current_screen;` we should use  these checks or use the `is_screen` method.


https://github.com/the-events-calendar/tribe-common/blob/master/src/Tribe/Admin/Helpers.php#L110-L125 

`Tribe__Admin__Helpers::instance()->is_screen( 'screen_id' )`

Reply:
Thanks for catching this one! The validation was applied